specification/ai/Azure.AI.Projects/connections/routes.tsp

```diff
@@ -0,0 +1,34 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Projects;
+
+alias ConnectionServiceTraits = SupportsClientRequestId &
+  NoRepeatableRequests &
+  NoConditionalRequests;
+
+alias ConnectionOperations = Azure.Core.ResourceOperations<ConnectionServiceTraits>;
+
+interface Connections {
+  @doc("Get a connection by name.")
+  get is ConnectionOperations.ResourceRead<Connection>;
```

Comment:
Missing a way to get credentials of a connection. We still need to discuss this.

Reply:
We discussed this and agree not to have credentials.